.github/workflows/tox.yml

```diff
@@ -30,3 +30,8 @@ jobs:
           sleep 5
       - name: Test with tox
         run: tox
+      - name: Run codacy-coverage-reporter
```

Comment:
is Codacy still active and monitoring this project? Remember we used that ages ago but at least all js repos migrated to CodeClimate.

Reply:
Yes. I found this out recently, too. Apparently the Codacy coverage report is embedded into what we run for tox. And Alex has managed to restore project tokens and to implement checks for the Python repos.

Reply:
ok, perfect, just wanted to make sure we do not send into some dead tool :-)